README.md

```diff
@@ -75,33 +75,24 @@ class MainActivity : AppCompatActivity(){
 ```
 
 
-Lastly, your view layer, in this case an activity, needs to tie into this system by subscribing to store updates and emitting actions whenever the app state needs to be changed:
+Lastly, your view layer, in this case an activity,
+needs to tie into this system by subscribing to store updates and
+emitting actions whenever the app state needs to be changed
+(assuming that `snake_case` View properties are coming from [Kotlin Android Extensions](https://kotlinlang.org/docs/tutorials/android-plugin.html)):
```

Comment:
snake_case is not a view shown in below example. Rename this to either button_up or button_down which is shown in example below.

Reply:
https://en.wikipedia.org/wiki/Snake_case